spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/NoOpSpanReporter.java

```diff
@@ -14,20 +14,18 @@
  * limitations under the License.
  */
 
-package org.springframework.cloud.sleuth.event;
-
-import org.springframework.cloud.sleuth.Span;
+package org.springframework.cloud.sleuth;
 
 /**
- * Emitted when a span was continued.
+ * Span reporter that does nothing
+ *
+ * @author Marcin Grzejszczak
  *
- * @author Spencer Gibb
+ * @since 1.0.0
  */
-@SuppressWarnings("serial")
-public class SpanContinuedEvent extends SpanContainingEvent {
+public class NoOpSpanReporter implements SpanReporter {
+	@Override
+	public void reportForCollection(Span span) {
```

Comment:
reportForCollection is unnecessarily long and maybe lies. Not all implementations will have a collection tier, ex what if it just reports to a log file? I'd simplify by just saying report